### DISCUSSION TITLE
IMMI paper for information on data structure

### DISCUSSION THREAD

**Discussion by hem-28** [BlueQuartzSoftware/DREAM3D#995] [General] [answered]
Hi,

I am trying to inform myself about the data structures in Dream3D. In the documentation, an IMMI paper is referenced, but the hyperlink takes me to  IMMI Journal homepage.  Could someone point me to the paper?

Thanks!

**Comment by imikejackson** [comment, marked as answer]
https://www.researchgate.net/publication/274363600_DREAM3D_A_Digital_Representation_Environment_for_the_Analysis_of_Microstructure_in_3D
